workflow/snakemake_rules/nextclade.smk

```diff
@@ -241,17 +257,17 @@ rule nextclade_info:
 
 
 rule combine_alignments:
-    message:
-        """
-        Generating full alignment by combining newly aligned sequences with previous (cached) alignment
-        """
+    """
+    Generating full alignment by combining newly aligned sequences with previous (cached) alignment
+    """
     input:
-        old_alignment=f"data/{database}/nextclade.aligned.old.fasta",
-        new_alignment=f"data/{database}/nextclade.aligned.upd.fasta",
+        nextclade_finished=f"data/{database}/nextclade.touch"
+        old_alignment=f"data/{database}/nextclade.{{seqtype}}.old.fasta",
     output:
-        alignment=f"data/{database}/aligned.fasta",
+        alignment=f"data/{database}/{{seqtype}}.fasta",
     params:
         keep_temp=config.get("keep_temp", "false"),
+        new_alignment=f"data/{database}/nextclade.{{seqtype}}.upd.fasta",
```

Comment:
Can this be an input rather than a param? It shares the same wildcards as the `old_alignment` so I don't think there should be an issue.

Reply:
Ah, I see. My question was answered in [Slack](https://bedfordlab.slack.com/archives/C01LCTT7JNN/p1686935320199049?thread_ts=1686858096.401379&cid=C01LCTT7JNN). This currently cannot be an input because it's not an output for the `run_nextclade` rule.